packages/eas-cli/src/commands/deploy/alias.ts

```diff
@@ -34,8 +34,8 @@ interface RawDeployAliasFlags {
 
 export default class WorkerAlias extends EasCommand {
   static override description = 'Assign deployment aliases';
-  static override aliases = ['deploy:alias', 'deploy:promote'];
-  static override state = 'beta';
+  static override aliases = ['worker:alias', 'deploy:promote'];
```

Comment:
Do we need to keep `worker:alias` around? I'd prefer to remove it, but we should technically keep it for backwards compatibility.

Reply:
I'd love to keep this. It doesn't really disrupt anything/anyone as opposed to potentially causing confusion if it goes missing. We can always remove it in a future major version